tools/sigh.js

```diff
@@ -216,7 +216,7 @@ async function tsc() {
 
 async function tslint(args) {
   let jsSources = [...findProjectFiles(process.cwd(), fullPath => {
-    if (/ts-build/.test(fullPath) || /server/.test(fullPath)) {
+    if (/ts-build/.test(fullPath) || /server/.test(fullPath) || /dist/.test(fullPath)) {
```

Comment:
This is starting to get tricky, separately we should at minimum pull this and below redundancy into a constant and document, and maybe refine the regex to include a path separator or be less generic (risk of mis-matching in future).

Reply:
Maybe move to a 'testPaths' mechanism like 'watchPaths'

Reply:
Perhaps include this config inside of package.json?

```
  "sigh": {
      "tslint-exclude": ["ts-build/**/*", "server/**/*", "dist/**/*"]
   }
```

WDYT?

Reply:
leave it as it is for now. We obviously need to have a conversation (as a team) about where overall build responsibilities should reside; there are clearly relative merits to sigh vs. npm but "both" isn't a good choice.

Reply:
(hmm, lintPaths, not testPaths...)